src/commands/install.yml

```diff
@@ -0,0 +1,64 @@
+description: >
+  Installs your project's node modules and cypress dependencies
+
+parameters:
+  install-command:
+    description: Overrides the default NPM command (npm ci)
+    type: string
+    default: ""
+  post-install:
+    description: >
+      Additional commands to run after running install
+      but before verifying Cypress and saving cache.
+    type: string
+    default: ""
+  cypress-cache-key:
+    description: Cache key used to cache the Cypress binary.
+    type: string
+    default: 'cypress-cache-{{ arch }}-{{ checksum "package.json" }}'
+  cypress-cache-path:
+    description: |
+      By default, this will cache the '~/.cache' directory so that the Cypress binary is cached. You can override this by providing your own cache path.
```

Comment:
```suggestion
      By default, this will cache the '~/.cache/Cypress' directory so that the Cypress binary is cached. You can override this by providing your own cache path.
```